src/openforms/registrations/contrib/objects_api/tests/test_objecttypes_client.py

```diff
@@ -0,0 +1,67 @@
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from ..client import get_objecttypes_client
+from ..models import ObjectsAPIConfig
+
+OBJECTTYPES_API_BASE_URL = os.getenv(
+    "OBJECTTYPES_API_BASE_URL", "http://localhost:8001/api/v2/"
+)
+# The default is the API key loaded in the docker service:
+OBJECTTYPES_API_KEY = os.getenv(
+    "OBJECTTYPES_API_KEY", "171be5abaf41e7856b423ad513df1ef8f867ff48"
+)
```

Comment:
with the docker-compose setup we have an env 100% under our control, so there is no reason to make this configurable through the environment. Just a constant is sufficient.